### ISSUE TITLE
Is the project is dead ?

### ISSUE THREAD

**Issue by ElderOrb** [Cutehacks/qpm#31]
May I ask about the current state of project? Is it still being developed?

**Comment by jrbarron** [comment]
@ElderOrb The project is not dead! 😄  When we started qpm we wanted to get a quick prototype out as soon as possible to see if there was any interest. It seems the tool and service definitely serve a purpose and after 9 months in the wild we have identified the things we want to fix for our 1.0 release. qpm in its current status is perfectly useable, but there are some warts and missing features we really want to fix and implement. Unfortunately we have been so busy with other projects that we haven't been dedicating much time to developing qpm. I have a local branch of changes that I have been working on as time permits (including merging some pull requests) so I can assure you that the project is not dead although I can certainly see why you would ask this question based on the lack of activity! Perhaps, I should write a blog post about our plans and try to set aside some time to actually write some code! :neckbeard:

By the way, a lot of the people contributing packages to qpm hang out on a Qt for Mobile Slack channel if you want to drop in and discuss things: http://slackin.qtmob.org
